packages/expo/hooks/post-publish.js

```diff
@@ -15,9 +15,29 @@ module.exports = async ({
   exp
 }) => {
   try {
-    log('beep boop')
-    log(url)
-    log(exp)
+    let apiKey
+    if (config && config.apiKey) {
+      apiKey = config.apiKey
```

Comment:
Discussed this further in person, but having the API key potentially in two places in `app.json` seems confusing, particularly when we can look up the key (and write it, via the CLI) in the default location. There should be just one place for the API key and the location in extras is already recommended/in use.